src/kube-runtime/src/plugins/teamwise_storage/storage_helper.py

```diff
@@ -174,31 +212,39 @@ def __get_azureblob_setup_commands(self, server_config, mount_point, relative_pa
                 "wget https://packages.microsoft.com/config/ubuntu/${release}/packages-microsoft-prod.deb",
                 "dpkg -i packages-microsoft-prod.deb",
                 "apt-get update",
-                "apt-get install --assume-yes blobfuse fuse", # blob to mount and fuse to unmount
+                "apt-get install --assume-yes blobfuse fuse",  # blob to mount and fuse to unmount
```

Comment:
why 2 spaces

Reply:
After I use [yapf](https://github.com/google/yapf), it will format code like this...

---
In reply to: [352492488](https://github.com/microsoft/pai/pull/3944#discussion_r352492488) [](ancestors = 352492488)

Reply:
Please refer [https://www.python.org/dev/peps/pep-0008/#inline-comments](https://www.python.org/dev/peps/pep-0008/#inline-comments)

---
In reply to: [353081445](https://github.com/microsoft/pai/pull/3944#discussion_r353081445) [](ancestors = 353081445,352492488)